src/lib.rs

```diff
@@ -369,9 +369,10 @@ pub mod formats {
         #[cfg_attr(docsrs, doc(cfg(feature = "bincode")))]
         pub type SymmetricalBincode<T, O = bincode_crate::DefaultOptions> = Bincode<T, T, O>;
 
-        impl<Item, SinkItem> Deserializer<Item> for Bincode<Item, SinkItem>
+        impl<Item, SinkItem, O> Deserializer<Item> for Bincode<Item, SinkItem, O>
         where
             for<'a> Item: Deserialize<'a>,
+            O: Options + Copy,
```

Comment:
Is `Copy` bound necessary?

Reply:
I think Clone is sufficient. Would that be ok? The serialize and deserialize methods take self by value.

Reply:
> The serialize and deserialize methods take self by value.

Ah, this is rather unfortunate. `Clone` would *probably* be better, yes.

Reply:
Done.